larq_compute_engine/tflite/python/interpreter.py

```diff
@@ -0,0 +1,57 @@
+from typing import List, Union
+
+import numpy as np
+from tqdm import tqdm
+
+from larq_compute_engine.tflite.python import interpreter_wrapper_lite
+
+__all__ = ["Interpreter"]
+
+
+class Interpreter(interpreter_wrapper_lite.LiteInterpreter):
+    """Interpreter interface for Larq Compute Engine Models.
+
+    !!! example
+        ```python
+        tflite_model = convert_keras_model(model)
+        interpreter = Interpreter(tflite_model)
+        interpreter.predict(input_data, verbose=1)
+        ```
+
+    # Arguments
+        flatbuffer_model: A serialized TFLite model in the flatbuffer format.
```

Comment:
I don't feel very strongly about this.

```suggestion
        lce_model = convert_keras_model(model)
        interpreter = Interpreter(lce_model)
        interpreter.predict(input_data, verbose=1)
        ```

    # Arguments
        flatbuffer_model: A serialized Larq Compute Engine model in the flatbuffer format.
```